docker/k8s/README.md

```diff
@@ -6,7 +6,8 @@ in SGX-SIM and SGX-HW mode.
 
 ## Prerequisites
 - kubectl-v1.15.2
-- a kubernetes cluster
+- a kubernetes cluster. [Minikube] (https://kubernetes.io/docs/setup/learning-environment/minikube/)
+  can be used to setup a single node kubernetes cluster.
```

Comment:
Change "kubernetes" to "Kubernetes" (and below at line 49)

Reply:
Done